test/jdk/java/lang/invoke/indify/Indify.java

```diff
@@ -23,124 +23,189 @@
+     */
     public String[] classpath = {"."};
+
+    /**
+     * Flag indicating whether to continue processing after encountering an error.
+     * Default is {@code false}.
+     */
     public boolean keepgoing = false;
+
+    /**
+     * Flag indicating whether to expand properties in input files.
+     * Default is {@code false}.
+     */
     public boolean expandProperties = false;
+
+    /**
+     * Flag indicating whether to overwrite existing files.
+     * Default is {@code false}.
+     */
     public boolean overwrite = false;
+
+    /**
+     * Flag indicating whether to suppress output messages.
+     * Default is {@code false}.
+     */
     public boolean quiet = false;
+
+    /**
+     * Flag indicating whether to enable verbose output.
+     * Default is {@code false}.
+     */
     public boolean verbose = false;
+
+    /**
+     * Flag indicating whether to process all items.
+     * Default is {@code false}.
+     */
     public boolean all = false;
+
+    /**
+     * Count of verify specifiers, with the default being -1 indicating no verification.
+     */
     public int verifySpecifierCount = -1;
 
+    public ClassModel classModel;
+
+    /**
+     * Processes command-line arguments to transform class files by incorporating JSR 292 features.
+     * <p>
+     * This method accepts various options and a list of files to be processed. If the '--java' option
+     * is specified, it runs the application with the provided arguments. Otherwise, it processes
+     * each file using the {@code indify} method.
+     * </p>
+     *
+     * @param av the command-line arguments
+     * @throws IOException if an I/O error occurs during file processing
+     * @throws IllegalArgumentException if the arguments are invalid
+     */
     public void run(String... av) throws IOException {
         List<String> avl = new ArrayList<>(Arrays.asList(av));
         parseOptions(avl);
-        if (avl.isEmpty())
+
+        if (avl.isEmpty()) {
             throw new IllegalArgumentException("Usage: indify [--dest dir] [option...] file...");
+        }
+
         if ("--java".equals(avl.get(0))) {
             avl.remove(0);
             try {
                 runApplication(avl.toArray(new String[0]));
             } catch (Exception ex) {
-                if (ex instanceof RuntimeException)  throw (RuntimeException) ex;
+                if (ex instanceof RuntimeException) {
+                    throw (RuntimeException) ex;
+                }
                 throw new RuntimeException(ex);
             }
             return;
         }
+
         Exception err = null;
         for (String a : avl) {
             try {
                 indify(a);
             } catch (Exception ex) {
-                if (err == null)  err = ex;
-                System.err.println("failure on "+a);
-                if (!keepgoing)  break;
+                if (err == null) {
+                    err = ex;
+                }
```

Comment:
Maybe use [Throwable​::addSuppressed​(Throwable)]:
```suggestion
                } else if (ex != err) {
                    err.addSuppressed(ex);
                }
```

[Throwable​::addSuppressed​(Throwable)]: https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java/lang/Throwable.html#addSuppressed(java.lang.Throwable)